### PR TITLE
Re-Added the Tidally Locked Rotation special with a population penalt…

### PR DESCRIPTION
**Pull request by Deepsloth**
…y and changed the penalty of the Eccentric Orbit special to a Supply minus. Includes (cleaned) AI updates by Dilvish.

### CODE REVIEW

**Comment by Deepsloth** [comment]
Updated version of #266. I cleaned the AI update by Dilvish and it works as far as i can tell.

Next Release Milestone.

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -5594,9 +5594,17 @@ ECCENTRIC_ORBIT_SPECIAL
 Eccentric Orbit
 
 ECCENTRIC_ORBIT_SPECIAL_DESC
-'''Population on the planet is decreased by 3, but [[encyclopedia RESEARCH_TITLE]] is increased by 3.
+'''Increases [[encyclopedia RESEARCH_TITLE]] by 3, but [[encyclopedia SUPLLY_TITLE]] is decreased by 2.
```
Comment (MatGB) [inline default/stringtables/en.txt]: Typo, "SUPLLY"

**Comment by Dilvish-fo** [comment]
I just wanted to confirm that yes, you were right about the correction you noted in # 266

**Comment by Vezzra** [comment]
Anything preventing this from getting merged?

**Comment by MatGB** [comment]
I thought it had been and would like it to be

**Pull request merged**